src/platform/packages/shared/kbn-alerts-as-data-utils/src/field_maps/legacy_experimental_field_map.ts

```diff
@@ -31,6 +32,12 @@ export const legacyExperimentalFieldMap = {
     required: false,
     array: true,
   },
+  [ALERT_GROUPING]: {
```

Comment:
I noticed that `kibana.alert.grouping` is added to other alerting indices too, not only in `.alerts-observability.threshold.alerts-default`. I believe that's intentional, right?

Reply:
Yes, that was the same for other fields that we've added as well.